lib/hanami/cli/commands/app/generate/struct.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "dry/inflector"
+require "dry/files"
+require "shellwords"
+require_relative "../../../naming"
+require_relative "../../../errors"
```

Comment:
```suggestion
require "dry/inflector"
require "dry/files"
require "shellwords"
require_relative "../../../naming"
require_relative "../../../errors"
```

I don't think we need any of these requires here anymore! Feel free to take them out of this as well as the `Generate::Operation` command before merging 😄

Reply:
Oh right, great! Gone